dayforce_client/client.py

```diff
@@ -99,16 +101,20 @@ class DayforceSFTP(object):
     username: str = attr.ib()
     password: str = attr.ib(repr=False)
     port: int = attr.ib(default=22)
-    known_hosts: str = attr.ib(default='~/.ssh/known_hosts')
+    # the host key can be retrieved with ssh-keyscan (pass just the base64 encoded part of the line beginning with ssh-rsa)
+    host_key: str = attr.ib(default=None)
```

Comment:
@bobzoller what are your thoughts on adding `repr` suppression in here so that the host key doesnt show in plain text when printing an object instance?

```suggestion
    host_key: str = attr.ib(repr=False, default=None)
```

Reply:
the host key is not secret, but it is a long ugly string... is that why you were thinking we should suppress it?

Reply:
@bobzoller whoops yeah you're right, not a secret. Let's leave it 👍